modules/node/init.zsh

```diff
@@ -6,13 +6,43 @@
 #   Zeh Rizzatti <zehrizzatti@gmail.com>
 #
 
+lazy_load_nvm() {
+  NVM_DIR=$1
+  # Skip adding binaries if there is no node version installed yet
+  if [ -d $NVM_DIR/versions/node ]; then
+    NODE_GLOBALS=(`find $NVM_DIR/versions/node -maxdepth 3 \( -type l -o -type f \) -wholename '*/bin/*' | xargs -n1 basename | sort | uniq`)
```

Comment:
I'd be willing to bet that this doesn't work in either osx or linux. There should be a way to use extended glob syntax to do something like this but only use zsh internals.

Reply:
After looking a bit closer, it looks fine. I've just run into strange issues with find on linux and osx being incompatible.